bin/wasabi.sh

```diff
@@ -265,22 +268,17 @@ package() {
 
   ./bin/fpm.sh -n ${name} -v ${version} -p ${profile}
 
+# FIXME: don't rebuild, cp dist/* target/*
   (cd modules/ui; \
     mkdir -p target; \
     for f in app bower.json Gruntfile.js constants.json karma.conf.js karma-e2e.conf.js package.json test .bowerrc; do \
```

Comment:
There's a test directory (`package.json test .bowerrc`), I think @felixgao wants to know if it's necessary to bundle it, i.e. having it inside the target dir.

I assume we don't need it since we don't test the ui, but if we would, we might?